### PR TITLE
Adds tests for feed-lightbox and feed-header component

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
**Changes proposed in this pull request**
- Added tests for feed-lightbox and feed-header component

**Link to live demo: ** 
http://pr-950-fossasia-loklaksearch.surge.sh

**Related to #414**

### CODE REVIEW

**Comment by shreyanshdwivedi** [comment]
@simsausaurabh  @praveenojha33 please review

**Review by praveenojha33 — APPROVED** [review]
LGTM :+1:

**Comment by shreyanshdwivedi** [comment]
@simsausaurabh please review

File: src/app/feed/feed-lightbox/feed-lightbox.component.spec.ts [hunk 1]
```diff
@@ -1,9 +1,79 @@
...
+				name : 'testUser'
+			},
+			text : 'testText',
+			retweet_count : 20,
+			favourites_count : 25,
+			created_at : new Date(),
+			videos : []
+		};
+		component.ttt();
+		expect(component.datetime).toBe('now');
```
Comment (simsausaurabh) [inline src/app/feed/feed-lightbox/feed-lightbox.component.spec.ts:58]: We actually need something meaningful in tests. Test cases should cover the actual codebase to test the important parts of project like API calls, data processing, etc. Only expecting each component and their parts to be ```truthy``` on project run is not something fruitful. Please try to add more meaningful cases/ add something which really tests something.

**Review by simsausaurabh — CHANGES_REQUESTED** [review]
Please see inline.

File: src/app/feed/feed-lightbox/feed-lightbox.component.spec.ts (hunk 1 above)
Comment (shreyanshdwivedi) [reply src/app/feed/feed-lightbox/feed-lightbox.component.spec.ts:58]: Will definitely try to add something more meaningful to this component

Comment (simsausaurabh) [reply src/app/feed/feed-lightbox/feed-lightbox.component.spec.ts:58]: @shreyanshdwivedi I am talking for overall in general, not only for this component.

**Review by simsausaurabh — APPROVED** [review]

**Comment by simsausaurabh** [comment]
@shreyanshdwivedi For your future PRs for increasing test coverage try only to add something meaningful to test since you have got familiar with writing test cases for the current codebase.

**Pull request merged**